mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.DeleteOrder = (req, res) => {
+  Order.remove({ _id: req.params.orderId }) 
```

Comment:
The function name in the controller is `DeleteOrder` but in the route, it's referred to as `removeOrder`. These names should be consistent to avoid confusion.